### PR TITLE
the login and registration for admin is done

### PR DESCRIPTION
**Pull request by mgemalma**
The registration and login is actually done for both users, however since there is no view for the normal user right now, after registration and login for a normal user nothing changes. If it's a admin user it switches to Alex's OwnerRestaurantListActivity.

### CODE REVIEW

**Comment by ashelleyPurdue** [comment]
For future reference, make more frequent, incremental commits.  It makes you life so much easier!  It'll help you break the work into incremental "steps", as well as giving you an easy way to remember where you left off when you take a break.  Just looking at your commit history can give you the needed context to get right back on the horse...as long as you commit frequently.

It also reduces your cognitive load while you're coding; if you need to undo a change you made, you don't need to remember all of the changes you made in order to make that happen; you can just do a git reset and everything will be undone.  If your commits are frequent, you won't any work you didn't mean to.

tl;dr: Commit after every small piece of work, and everything will be easier.
